randovania/games/am2r/assets/migration_data.json

```diff
@@ -0,0 +1,96 @@
+{
+    "remove_teleporternode_data": {
```

Comment:
Skip this file. Let's avoid having to worry about migrating preset/rdvgame for a game that is quite new